lib/matplotlib/figure.py

```diff
@@ -173,77 +173,81 @@ def __init__(self, left=None, bottom=None, right=None, top=None,
                  wspace=None, hspace=None):
         """
         All dimensions are fractions of the figure width or height.
-        Defaults are given by :rc:`figure.subplot.[name]`.
+        Defaults are given by :rc:`figure.subplot.*`.
```

Comment:
would leave `[name]`

Reply:
I don't really like any of them but I don't know a better one. I think that `figure.subplot.[name]` make it easier to understand what to write.